sdks/java/testing/test-utils/src/main/java/org/apache/beam/sdk/testutils/metrics/IOITMetrics.java

```diff
@@ -70,4 +72,21 @@ public static void publish(
     }
     ConsoleResultPublisher.publish(results, uuid, timestamp);
   }
+
+  public void publishToInflux(final InfluxDBSettings settings) {
+    MetricsReader reader = new MetricsReader(result, namespace);
+    Collection<NamedTestResult> namedTestResults = reader.readAll(metricSuppliers);
+
+    InfluxDBPublisher.publishWithSettings(namedTestResults, settings);
+  }
+
+  public static void publishToInflux(
+      final String uuid,
+      final String timestamp,
+      final Collection<NamedTestResult> results,
+      final InfluxDBSettings settings) {
+
+    InfluxDBPublisher.publishWithSettings(results, settings);
+    ConsoleResultPublisher.publish(results, uuid, timestamp);
```

Comment:
For the time being this is unnecessary, the results will be published to the console twice, since BQ publisher does the same thing.

Reply:
👍